python/tskit/drawing.py

```diff
@@ -221,6 +246,153 @@ def remap_style(original_map, new_key, none_value):
         return str(text_tree)
 
 
+def draw_tikz(
+    tree,
+    node_labels=None,
+    tree_height_scale=None,
+    order=None,
+    aspect=None,
+    scale=None,
+    style=None,
+    standalone=False,
+):
+    """
+    Return a string containing latex/tikz commands to draw a tskit.Tree.
+
+    Save the string to a file (e.g. tree.tex) and include it in your latex
+    document with ``\input{tree.tex}``.
+    """
+    if node_labels is None:
+        node_labels = {node: str(node) for node in tree.nodes()}
+    if tree_height_scale is None:
+        tree_height_scale = "rank"
+    if aspect is None:
+        aspect = 1
+    if style is None:
+        style = ""
+    else:
+        # hack to avoid empty line in tikzpicture options (which causes problems)
+        style = "\n" + style
+    order = check_order(order)
+
+    template = textwrap.dedent(
+        r"""
+        \begin{tikzpicture}[
+            scale=$scale,
+            edge/.style = {
+                draw=black,
+            },
+            node/.style = {
+                circle,
+                fill=black,
+                line width=0,  % border
+                % radius
+                minimum size=2,
+                inner sep=0,
+            },
+            node text/.style = {
+                %font=\bfseries,
+            },
+            left node text/.style = {
+                node text,
+                above left=.1,
+            },
+            right node text/.style = {
+                node text,
+                above right=.1,
+            },
+            leaf node text/.style = {
+                node text,
+                below=.1,
+            },
+            % User customisation goes here. $user_style_text
```

Comment:
Did you mean to have $user_style_text on a comment line?

Reply:
Yeah, I made an arbitrary choice about how to deal with something annoying in the required syntax. The `options` part of the tikzpicture (the bit inside `[` and `]`) doesn't seem to allow empty lines. If `$user_style_text` is on its own line, then this is a syntax error when the user doesn't specify anything. So I just prepend "\n" to the style when the user *does* specify something. This is ugly, and I'm sure we can do better.

Reply:
Sounds good - needs a bit of explanation somewhere, though, as this is the sort of thing someone would "fix" while doing a pass through the code.